### DISCUSSION TITLE
Unable to find lambda for route: /sitemap.xml

### DISCUSSION THREAD

**Discussion by MarcusIsh** [vercel/community#5028] [Help]
### Summary

I have taken over a project from a client and we are trying to get our git branches to preview for this client to view the progress, however we are getting this error from vercel deployment after the rest of the application is built. I am new to vercel, but it is happening on every branch(havent tried on master as we are not pushing to master yet)

I would like to mention that when we took over this project from the previous developer, we changed the git repo in vercel.

Traced Next.js server files in: 4.098s
--
10:29:23.873 | Created all serverless functions in: 8.827s
10:29:23.876 | Error: Unable to find lambda for route: /sitemap.xml

any help to get our branches previewing would be greatly appreciated 

I would also like to mention that i did update the current sitemap.xml that is in the root app folder, however there is not change. The site map is in the project and does exist.

### Steps to Reproduce

1. Push edited code to github
2. vercel takes the deployment and starts to build
3. fails on build

**Comment by amyegan** [comment]
Hi @MarcusIsh! I need some more information to be able to determine what's causing the problem. Do you have a [minimal reproducible example](https://vercel.com/support/articles/creating-a-minimal-reproducible-example) that I can use to help with debugging?
